### PR TITLE
Feature/request form

### PR DESCRIPTION
**Pull request by JohnLeeroy**
Added the basic UI & hooks for the request form.  
We are missing the API hooks & handling configuration changes.

### CODE REVIEW

File: app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java [hunk 1]
```diff
@@ -40,10 +39,20 @@
     public static final String EXTRA_LOCATION = "location";
     public static final String EXTRA_TREETYPE = "type";
 
+    FloatingActionButton fab;
```
Comment (LondonAtlas) [inline app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java:49]: could we change fab to floatingActionButton or floatingButton?

File: app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java [hunk 2]
```diff
@@ -105,7 +114,7 @@ public void onCameraChange(CameraPosition cameraPosition) {
         try {
             SavedTreesProvider provider = SavedTreesProvider.getInstance(getApplicationContext());
             VisibleRegion vr = map.getProjection().getVisibleRegion();
-            addMarkersToMap(provider.getVisibleTrees(vr));
+            addMarkersToMap(provider.getVisibleTrees(vr, 20));
```
Comment (LondonAtlas) [inline app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java]: What does this magic 20 do?

File: app/src/main/java/com/codefororlando/streettrees/activity/RequestTreeActivity.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                break;
+            default:
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    @Override
+    public void next() {
+        int currentItemIndex = listPager.getCurrentItem();
+        if(currentItemIndex < (5)) {
```
Comment (LondonAtlas) [inline app/src/main/java/com/codefororlando/streettrees/activity/RequestTreeActivity.java]: (5)?

**Review by LondonAtlas — APPROVED** [review]

File: app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java (hunk 1 above)
Comment (JohnLeeroy) [reply app/src/main/java/com/codefororlando/streettrees/activity/MainActivity.java:49]: no

**Review by LondonAtlas — APPROVED** [review]

**Pull request merged**